admins.py

```diff
@@ -1,25 +1,61 @@
 #!/usr/bin/env python
 import urllib
+import argparse
+import logging
 
 import requests
 
 from client import get_token
 
-if __name__ == '__main__':
+logger = logging.getLogger(__name__)
+
+def check_admins(org):
     headers = {
-        'Accept': 'application/vnd.github.moondragon+json',
+        # new api requires new accept per
+        # https://developer.github.com/changes/2015-06-24-api-enhancements-for-working-with-organization-permissions/#preview-period
+        'Accept': 'application/vnd.github.ironman-preview+json',
         'Authorization': 'token %s' % get_token()
     }
     params = {'filter': '2fa_disabled',
               'role': 'admin'}
 
     admins_with_1fa_url = '%s?%s' % (
-        'https://api.github.com/orgs/%s/members' % 'mozilla',
+        'https://api.github.com/orgs/%s/members' % org,
         urllib.urlencode(params)
     )
     resp = requests.get(admins_with_1fa_url, headers=headers)
+    if resp.status_code not in [200,]:
```

Comment:
nit, this is a list for no good reason, use (immutable) tuples unless you plan on changing the sequence ;)